package.json

```diff
@@ -1,67 +1,34 @@
 {
-  "name": "mobi.css",
-  "version": "2.0.0-beta.1",
-  "description": "A themeable, pluginable, mobile-first css framework",
-  "main": "index.scss",
-  "scripts": {
-    "start": "gulp",
-    "build": "gulp build",
-    "test": "npm run lint",
-    "lint": "stylelint **/*.scss --syntax scss && eslint .",
-    "preversion": "npm test",
-    "version": "npm run build && git add -A ."
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mobi-css/mobi.css.git"
-  },
-  "keywords": [
-    "mobi-css",
-    "css-framework",
-    "css",
-    "sass",
-    "scss",
-    "design",
-    "mobile-first",
-    "flexbox"
-  ],
-  "author": "xcatliu <xcatliu@gmail.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/mobi-css/mobi.css/issues"
-  },
-  "homepage": "http://getmobicss.com",
-  "devDependencies": {
-    "autoprefixer": "^6.4.0",
-    "debounce": "^1.0.0",
-    "ecstatic": "^2.1.0",
-    "eslint": "^3.4.0",
-    "eslint-config-airbnb-base": "^5.0.3",
-    "eslint-plugin-import": "^1.14.0",
-    "gulp": "^3.9.1",
-    "gulp-clean-css": "^3.0.3",
-    "gulp-insert": "^0.5.0",
-    "gulp-postcss": "^6.1.1",
-    "gulp-rename": "^1.2.2",
-    "gulp-sass": "^2.3.2",
-    "gulp-sourcemaps": "^1.6.0",
-    "pagic": "^0.6.0",
-    "rimraf": "^2.5.4",
-    "run-sequence": "^1.2.2",
-    "stylelint": "^7.9.0",
-    "stylelint-config-standard": "^16.0.0"
-  },
-  "dependencies": {
-    "mobi-plugin-button": "^1.0.0",
-    "mobi-plugin-container": "^1.0.0",
-    "mobi-plugin-flexbox": "^1.0.0",
-    "mobi-plugin-form": "^1.0.0",
-    "mobi-plugin-reset": "^1.0.0",
-    "mobi-plugin-scroll-view": "^1.0.0",
-    "mobi-plugin-show-hide-on-mobile": "^1.0.0",
-    "mobi-plugin-table": "^1.0.0",
-    "mobi-plugin-text": "^1.0.0",
-    "mobi-plugin-top-gap": "^1.0.0",
-    "mobi-theme-base": "^1.0.0"
-  }
+	"scripts": {
+		"bootstrap": "lerna bootstrap --hoist",
+		"build": "lerna run build"
+	},
+	"browserslist": [
+		"last 5 iOS versions",
+		"last 5 Android versions",
+		"last 5 ChromeAndroid versions",
+		"last 5 UCAndroid versions",
+		"last 5 FirefoxAndroid versions",
+		"last 5 OperaMini versions",
+		"last 5 OperaMobile versions",
+		"last 5 Samsung versions",
+		"last 3 Chrome versions",
+		"last 3 Firefox versions",
+		"last 3 Edge versions",
+		"last 3 Opera versions"
+	],
+	"devDependencies": {
+		"cssnano": "^4.0.0-rc.0",
+		"gulp": "^3.9.1",
+		"gulp-insert": "^0.5.0",
+		"gulp-postcss": "^7.0.0",
+		"gulp-rename": "^1.2.2",
+		"gulp-sourcemaps": "^2.6.1",
+		"lerna": "^2.1.2",
+		"postcss-cssnext": "^3.0.2",
+		"postcss-import": "^10.0.0",
+		"rimraf": "^2.6.1",
+		"stylelint": "^8.0.0",
+		"stylelint-config-standard": "^17.0.0"
```

Comment:
missing author, repo, license info which a npm install will generate warnings.

Reply:
shouldn't we add lerna a dev dependency?

Reply:
Fixed, thanks!